opendevin/llm/readers.py

```diff
@@ -0,0 +1,523 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import base64
+import json
+import subprocess
+import zipfile
+from abc import ABC, abstractmethod
+from pathlib import Path
+from typing import Any, Union
+
+import charset_normalizer
+import cv2
+import docx
+
+# import markdown
```

Comment:
Dead code

```suggestion
```

Reply:
nvm I see markdown might still be useful. It looks like it's just temporarily commented out.